packages/lit-element/CHANGELOG.md

```diff
@@ -55,7 +55,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - [Breaking] Removed build support for TypeScript 3.4.
 - [Breaking] Decorators are no longer exported from the `lit-element` module. Instead, import any decorators you use from `lit-element/decorators/*`.
 - [Breaking] `lit-html` has been updated to 2.x.
-- [Breaking] Support for running in older browsers has been removed from the default configuration. Import the `platform-support` module to support Shady DOM. Note also that Lit parts inside `<style>` elements are no longer supported.
+- [Breaking] Support for running in older browsers has been removed from the default configuration. Import the `polyfill-support` module to support Shady DOM. Note also that Lit parts inside `<style>` elements are no longer supported.
```

Comment:
I'd add a new item and a (Since... ) note and we can collapse these at release

Reply:
Done.